portal-impl/src/com/liferay/portal/deploy/hot/HookHotDeployListener.java

```diff
@@ -2322,6 +2320,7 @@ private void _initServices(
 	};
 
 	private static final String[] _PROPS_VALUES_OBSOLETE = {
+		"field.enable.com.liferay.portal.kernel.model.Contact.male",
```

Comment:
I'm not sure if there is supposed to be complete parity between legacy properties and this constant.  If so then yes we should add them here, but otherwise I only moved the property found in the constants above.